src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,106 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double INCREASE_COEFFICIENT = 1.5;
+    private static final double EMPTY_SIZE = 0;
```

Comment:
In my opinion its redudant variable, because you can check isEmpty, by comparing with 0, it`s simplier